tools/testdata-scripts/src/bin/make-testdata-legacy.rs

```diff
@@ -19,31 +20,17 @@ fn main() {
 
     let data_root = Path::new(concat!(
         core::env!("CARGO_MANIFEST_DIR"),
-        "/../../provider/datagen/tests/data/"
+        "/../../provider/testdata/data/"
     ));
 
+    std::fs::create_dir_all(data_root).unwrap();
+
     let source = SourceData::offline()
-        .with_cldr(data_root.join("cldr"), Default::default())
-        .unwrap()
-        .with_icuexport(data_root.join("icuexport"))
+        .with_cldr_latest(Default::default())
         .unwrap()
-        .with_segmenter_lstm(data_root.join("lstm"))
+        .with_icuexport_latest()
```

Comment:
Question: Why are you changing this script to read from release data instead of repo data? I guess it's fine

Reply:
I remember some of my reviewers not liking relative paths.